### PR TITLE
Add service mesh switcher

### LINKED ISSUE #1988: [Learn] Add service mesh switcher for each course
**Current State:**
A user can take a course under a learning path from a variety of service meshes available. But, it lacks the switcher to do so.

**Desired State:**
A section needs to be added under the TOC on each chapter page, which will allow the user to change the service mesh using which they want to take the course. Like the `Service meshes:` section we have in the image below:
Screenshot from 2021-07-12 19-30-55
The working of the same can be understood from this [page](https://storybook.js.org/tutorials/intro-to-storybook/react/en/get-started/).

---
**Contributor Resources**
The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- Check the Learn Layer5 site [Design Spec](https://docs.google.com/document/d/1tQ9tLbetBVKP7z76FR19vsxbAfoFHZzZYxCt7lJpAhU/edit) to know more about the site structure and development plans.
- Checkout the Learn Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/?node-id=4853%3A0).

### PR DESCRIPTION
**Pull request by sudo-NithishKarthik**
https://user-images.githubusercontent.com/75248557/127768164-654a4b0a-05d3-4266-b4b0-01d4439e7bf0.mov

This PR fixes #1988

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 18cc38df7564571b10658a6be3b529e0edfe94be at: https://61067eedf4f7e83d89d1c396--layer5ng.netlify.app

File: src/sections/Learn-Layer5/Chapters/index.js [hunk 1]
```diff
@@ -3,17 +3,72 @@ import { MDXRenderer } from "gatsby-plugin-mdx";
...
+
+
+  
+
   return (
     <ChapterWrapper>
       <Container className="chapter-container">
         <Row>
           <Col sm={12} md={3}>
             <TOC courseData={courseData} chapterData={chapterData} location={location} />
```
Comment (Nikhil-Ladha) [inline src/sections/Learn-Layer5/Chapters/index.js:65]: We need to update this component as well, to update the paths in the TOC, for the selected service mesh. An `useState` defined in this component can help us achieve that.

Comment (sudo-NithishKarthik) [reply src/sections/Learn-Layer5/Chapters/index.js:65]: I see, i missed it, will do

**Comment by l5io** [comment]
🚀 Preview for commit e07dafc9d26df6433caa605061ec48a2bafed1af at: https://6107da6b42afb7f801a2519b--layer5ng.netlify.app

**Comment by sudo-NithishKarthik** [comment]
@Nikhil-Ladha , do i have to add anything else, if not, it can be merged ig

**Comment by leecalcote** [comment]
The Learning Paths are looking great.

**Comment by Nikhil-Ladha** [comment]
I think the fallback mechanism can be added in a separate PR, let's get this is in.
Do you want to take a jab at that @sudo-NithishKarthik or shall I take it?

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by sudo-NithishKarthik** [comment]
Sure, i'll do it, ig it won't take much time 👍🏽

**Comment by Nikhil-Ladha** [comment]
Okay, so while you take a look into that, do you mind updating the `Service Meshes You can Learn` section on the course overview page to highlight the service meshes available for that course only.
**Note:** That shouldn't be switcher but only icons for service mesh along with their names.

**Comment by sudo-NithishKarthik** [comment]
Got it

**Pull request merged**